test_scripts/Defects/7_1/885_Alert.lua

```diff
@@ -0,0 +1,88 @@
+---------------------------------------------------------------------------------------------------
+-- Issue: https://github.com/SmartDeviceLink/sdl_core/issues/885
+---------------------------------------------------------------------------------------------------
+-- Description: Check that SDL stops processing softButton events once a response is received for the RPC.
+-- 'Alert' scenario.
+--
+-- Steps:
+-- 1. App is registered and activated
+-- 2. App sends 'Alert' RPC with soft buttons
+-- SDL does:
+--  - transfer request to HMI
+-- 3. HMI sends 'OnButtonEvent' and 'OnButtonPress' notifications
+-- SDL does:
+--  - transfer notifications to the App
+-- 4. HMI provides a response for 'Alert'
+-- SDL does:
+--  - transfer response to the App
+-- 5. HMI sends 'OnButtonEvent' and 'OnButtonPress' notifications
+-- SDL does:
+-- - not transfer notifications to the App
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('user_modules/sequences/actions')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local expected = {
+  yes = true,
+  no = false
+}
```

Comment:
Instead of using a table, we could have `expected` as a boolean value
```suggestion
local expected = true
```
and use 
```
sendOnButtonEvents(expected)
```
for the first function call and

```
sendOnButtonEvents(not expected)
```
for the second function call

Reply:
@ShobhitAd Please find update in f8a1df3